frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1317,6 +1344,28 @@ export const funnelLogic = kea<funnelLogicType<openPersonsModelProps>>({
             }
         },
     }),
+    urlToAction: ({ actions }) => ({
```

Comment:
If we keep `funnel_advanced` in filters and don't persist it into a separate value, we might be able to get rid of a lot of boilerplate here.